src/map.h

```diff
@@ -1583,6 +1583,10 @@ class map
  * Used in determining speed of temperature change of items
  */
 unsigned int temp_difference_ratio( int temp_one, int temp_two );
+/**
+ * Returns freezing point temperature of item, mainly for non-water freezing points
+ */
+int check_freezing_temperature( item &it );
```

Comment:
Why is this in the public interface of the `map` class? There's nothing map related in there. Moreover, the absence of `const`s implies that both `item` and `map` itself can be modified by the function.

Reply:
Ok, just though you want me to push them out to a separate file.